sktime/forecasting/base/_base.py

```diff
@@ -68,6 +72,11 @@ def __init__(self):
         self._fh = None
         self._cutoff = None  # reference point for relative fh
 
+        self.converter_store = dict()  # storage dictionary for input/output conversions
+
+        # "safe" initialization in case fit is overridden
```

Comment:
`fit` should not be overwritten anymore, according to the new extension guidelines.

The safe initialization is for downwards compatibility currently.
Going forward, we may want to think of a set of informative error messages if extender users overwrite `fit` accidentally.

What do you propose as a concrete change?

Reply:
I see! Just to check, you have this safe initialisation because the refactoring of forecasters is not yet complete, and you intend to remove it once the refactoring is complete?

Reply:
superseded, `y_in_type` is now entirely removed since we removed output conversions.